api/src/test/resources/application.properties

```diff
@@ -9,4 +9,7 @@ spring.integration.jdbc.initialize-schema=always
 spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 spring.jpa.hibernate.ddl-auto=create-drop
 
+#-- disable liquibase for junit test which uses H2 instead of postgres
```

Comment:
This is probably OK for now, as long as Liquibase migrations merely follow the domain model changes (in which case both running Liquibase or using Hibernate Auto-DDL would lead to the same schema). But if we ever add a migration that adds something to the DB that is *not* expressed in the model, we might have to start running Liquibase in tests (or think of a more creative solution). Just a thought at this point

Reply:
I concur